srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/ValueRetriever.java

```diff
@@ -22,6 +24,12 @@ public void execute(final Patient patient, final VariableEntry variableEntry, fi
         {
             variableEntry.getDynamicValues().put(key, patient.getGender());
         }
+
+        @Override
+        public String retrieveReferenceInfo(final Patient patient, final Variable var)
```

Comment:
Man, why don't you just have the base class define this to return an empty string and only override as needed?